### PR TITLE
Add rainy and snowy backgrounds.

### PR DESCRIPTION
**Pull request by t1chita**
backgrounds are ready to use but, i need to change colors of snowFall and clouds (SpriteKIT's Partice Files) texture.they are not supporting dark and light modes now.

### CODE REVIEW

File: HW37-Tsvimify-Collaboration/Backgrounds/RainyAndSnowyBackground.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    
+    @Environment(\.colorScheme) var colorScheme
+    let sksFileName: String
+    let backgroundColorTop: Color
+    let backgroundColorBottom: Color
+    
+    var body: some View {
+        ZStack {
+            Group {
+                SpriteView(scene: RainAndSnowAnimations(sksFileName: sksFileName),options: [.allowsTransparency])
```
Comment (Tornike222) [inline HW37-Tsvimify-Collaboration/Backgrounds/RainyAndSnowyBackground.swift]: use Computed Property there and below

**Review by Tornike222 — CHANGES_REQUESTED** [review]
Great work 🚀 just make a separated views with computed properties

File: HW37-Tsvimify-Collaboration/SpriteKitFiles/ParticeFiles/Assets.xcassets/Particle Sprite Atlas.spriteatlas/Contents.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (irinkadat) [inline HW37-Tsvimify-Collaboration/SpriteKitFiles/ParticeFiles/Assets.xcassets/Particle Sprite Atlas.spriteatlas/Contents.json]: This `assets` here isn't necessary for the project

**Review by irinkadat — APPROVED** [review]
Great work on this feature! The code looks clean and well-organized.

**Review by Tornike222 — APPROVED** [review]
Great, You have written well structured code

**Pull request merged**